### ISSUE TITLE
Support for partial file download

### ISSUE THREAD

**Issue by Twv** [dropbox/dropbox-sdk-obj-c#58]
I've read through the API documentation (and github issues), but cannot find a way to create a request for partial file download (e.g. first 10.000 bytes).
I've only found how to download the whole file:

``` objectivec
DBDownloadDataTask *dTask = [client.filesRoutes downloadData:file];
```

Is there a way (maybe I have missed it) to only download a part of the file using the API?
I know it is possible to make requests for file parts using REST calls, but then I cannot use the objc API.

Any help would be appreciated.

BR, Jan.

**Comment by greg-db** [comment]
This isn't implemented in the SDK unfortunately, but we'll consider it a feature request.

**Comment by sanhue0302** [comment]
Will this feature add on next release?

**Comment by greg-db** [comment]
I can't make any promises, but I've sent this along as a feature request.

**Comment by scobbe** [comment]
@Twv @sanhue0302: This is fixed now in [v3.0.3](https://github.com/dropbox/dropbox-sdk-obj-c/releases/tag/3.0.3) of the SDK. Apologies for the long delay.

**Comment by greg-db** [comment]
For reference, you can use this functionality via the download methods with `:byteOffsetStart:byteOffsetEnd:`, such as [-downloadData:byteOffsetStart:byteOffsetEnd:](https://dropbox.github.io/dropbox-sdk-obj-c/api-docs/latest/Classes/DBFILESRoutes.html#/c:objc(cs)DBFILESRoutes(im)downloadData:byteOffsetStart:byteOffsetEnd:).
